package.json

```diff
@@ -0,0 +1,53 @@
+{
+    "name": "@singleton-i18n/g11n-js-sdk",
+    "version": "0.1.0",
+    "main": "./bundles/g11n-js-sdk.umd.js",
+    "description": "This is I18n level 2 common library for javascript based clients like g11n-angular-client,g11n-js-client",
+    "author": "Hong Zhang(hongzhang@vmware.com)",
+    "license": "EPL-2.0",
+    "module": "./dist/index.js",
+    "typings": "./dist/index.d.ts",
+    "scripts": {
+        "compile": "rimraf dist && tsc -p ./ --outDir dist/",
+        "build": "webpack",
+        "prepublish": "rimraf dist && tsc -p ./ --outDir dist/ && npm run build",
+        "jest": "jest -c jest.config.json",
+        "test": "npm run jest"
+    },
+    "keywords": [
+        "singleton",
+        "i18n",
+        "l10n",
+        "node",
+        "VIP"
```

Comment:
It's better remove 'VIP' here

Reply:
Ok, I'll remove it next time is there are no other issues currently.

Reply:
https://github.com/hongzhang-github/singleton/commit/d852d11209b1ce8dd0e2e7d2307d3b458a6217b7